ecasbot/__init__.py

```diff
@@ -429,8 +429,20 @@ def handle_msg(message) -> None:
             """
             try:
                 # Removing messages from restricted members...
-                if self.__check_message_entities(message) or self.__check_message_forward(
-                        message) or self.__check_message_spam(message):
+                entities = self.__check_message_entities(message)
+                forward = self.__check_message_forward(message)
+                spam = self.__check_message_spam(message)
+                author_name = "{} - {} {}".format(message.from_user.id,
+                                                  message.from_user.first_name, message.from_user.last_name)
+
+                self.__logger.debug("chat: {} ({}) ".format(message.chat.id, message.chat.title))
+                self.__logger.debug("author: {} ".format(author_name))
+                self.__logger.debug("message: {} ".format(message.text))
+                self.__logger.debug("entities: {}".format(entities))
+                self.__logger.debug("forward: {}".format(forward))
+                self.__logger.debug("spam: {}".format(spam))
+
+                if entities or forward or spam:
```

Comment:
Logger need to be pasted after this row to log messages will be removed by bot.

Reply:
This is intentional - I'd like to log messages from all restricted users so that later on I could tweak bot's URL regexp and /or introduce additional checks.

It uses `debug` so it won't be logged to stdout/stderr